### ISSUE TITLE
Missing strings in the Base Menu file

### ISSUE THREAD

**Issue by Vegebutcher** [typora/Typora-i18n#133]
No original lines for "Copy as Plain Text" in Edit menu and "Check Updates" in Help.

BTW, "Start/Stop Speaking" entries in Edit menu (lines 99 & 100), as well as Format>Image>Copy Image to... (line 285) although translated, don't appear translated in the menu.

**Comment by Vegebutcher** [comment]
> `Copy Image to` exists in menu strings.

Yep, it does, but it's displayed in English in the application, although it is translated. The same applies to Start/Stop Speaking entries:

![copy image to](https://user-images.githubusercontent.com/907743/66963463-0ad34000-f074-11e9-8ac1-1f46d3e62a91.png)
![speaking](https://user-images.githubusercontent.com/907743/66963464-0ad34000-f074-11e9-945b-88e2f4c10250.png)

**Comment by abnerlee** [comment]
use `Copy Image to…` not `Copy Image to...`

**Comment by Vegebutcher** [comment]
Both lines - 240 and 285 - are translated and both have single `…` character in the original string. Where did you find `...`?
By the way, `Start/Stop Speaking` are already in the Edit menu section (lines 99 and 100), so why are they repeated in the new commit (lines 292 and 293)?
